xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -90,23 +106,26 @@ namespace XBMCAddon
-      /// Generate a key request, used for request/response exchange between the app and a license server
-      /// to obtain or release keys used to decrypt encrypted content.
-      /// After the app has received the key request response from the server,
-      /// it should deliver to the response to the MediaDrm instance using 
-      /// the method provideKeyResponse, to activate the keys.
-      ///
-      /// @param      [byte] init     Initialization bytes container-specific data,
-      ///                             its meaning is interpreted based on the mime type provided 
+      /// Generate a key request, used for request/response exchange between the app
+      /// and a license server to obtain or release keys used to decrypt encrypted content.
+      /// After the app has received the key request response from the license server,
+      /// it should deliver to the response to the %DRM instance using
+      /// the method \ref xbmcdrm_ProvideKeyResponse "ProvideKeyResponse", to activate the keys.
+      ///
+      /// @param      init            byte - Initialization bytes container-specific data,
+      ///                             its meaning is interpreted based on the mime type provided
       ///                             in the mimeType parameter. It could contain, for example,
-      ///                             the content ID, key ID or other data required in generating the key request.
-      /// @param      String mimeType Type of media which is xchanged, e.g. application/xml, video/mp4
-      /// @param      bool offlineKey Specifes the type of the request.
+      ///                             the content ID, key ID or other data required in generating
+      ///                             the key request.
+      /// @param      mimeType        string - Type of media which is exchanged
+      ///                             (e.g. "application/xml", "video/mp4")
+      /// @param      offlineKey      bool - Specifes the type of the request.
       ///                             The request may be to acquire keys for Streaming or Offline content
-      /// @param      [map] optionalParameters optional Will be included in the key request message to allow a client application 
-      ///                                               to provide additional message parameters to the server
+      /// @param      optionalParameters [opt] map - Will be included in the key request message
+      ///                                to allow a client application to provide additional 
+      ///                                message parameters to the server
       ///
-      /// @return     The opaque key request data (challenge) which is send to key server
+      /// @return     byte - The opaque key request data (challenge) which is send to key server
       ///
       ///------------------------------------------------------------------------
       /// @python_v18 New function added.
```

Comment:
`@python_v19 With python 3 the init param must be a bytearray instead of byte.`

Reply:
done